### PR TITLE
delete JWT_secret related parts in README.md

### PR DESCRIPTION
**Pull request by hadley0828**
## What's [new / improved / fixed]

- Remove JWT_Secret from docs

## Details

- Related issue: #306 

## Upgrading

Instructions for other developers on how to upgrade their environment after
pulling this new code. Please tick (i.e. put an 'x' in) the boxes for the
actions that are necessary.

- [ ] Run GraphQL Codegen in the [frontend/actions service/playout service/realtime service]
- [ ] Re-install NPM packages in [insert name of folder]
- [ ] Apply Hasura migrations
- [ ] Apply Hasura metadata
- [ ] Update Auth0 rules
- [ ] Update environment variables
- [ ] Re-deploy AWS CDK [and update AWS environment variables]

## Deployment

Instructions for how to deploy these changes to production. Please tick (i.e.
put an 'x' in) the boxes for the actions that are necessary.

- [ ] Apply migrations to Hasura
- [ ] Apply metadata to Hasura
- [ ] Reload enum table/values in Hasura for [table names]
- [ ] Re-deploy frontend
- [ ] Re-deploy actions service
- [ ] Re-deploy playout service
- [ ] Re-deploy real-time service
- [ ] Flush real-time service state (Redis/RabbitMQ)
- [ ] Update environment variables for [name of component]

Any other steps necessary to re-deploy?

### CODE REVIEW

**Review by EdNutting — CHANGES_REQUESTED** [review]
Looks good so far. I think there's also a reference in `hasura/README.md:15` that needs removing too.

**Review by EdNutting — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -549,19 +549,6 @@ created above.
...
-1. Copy your key (the whole JSON object that is generated in the JWT Config
-   box) into your local `hasura/.env.local` file
-   - e.g. `HASURA_GRAPHQL_JWT_SECRET='your key goes in here'`
-   - **Don't forget the wrapping single quotes!**
-1. Uncomment the `HASURA_GRAPHQL_JWT_SECRET: ${HASURA_GRAPHQL_JWT_SECRET}`
-   line in `docker-compose.yaml`
-   - Don't forget to restart the `Hasura Console -- Local Development` task
-     in VSCode! (Again, if you get an error message about `version check: failed to get version from server: failed making version api call...`
-     try running the task again -- this could be due to a race condition.)
 1. Optionally: Copy your key into Hasura Cloud Environment configuration
```
Comment (EdNutting) [inline README.md:552]: Come to think  of it, this bullet point should be deleted too - and in fact, the heading `#### 7. Configure Hasura` can then be deleted as well. Please update the numbering of subsequent headings (8, 9, 10)

**Review by EdNutting — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hadley0828) [reply README.md]: I have deleted this whole section and changed the order number accordingly. What's more, for Create Rules section, do we need to make some change to `Name it something like Hasura JWT` ?

**Review by EdNutting — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (EdNutting) [reply README.md]: It's still required to follow the Hasura keys specification so I think the name of the rule remains appropriate.

**Comment by EdNutting** [comment]
I think this is probably ready to merge. @rossng any comments?

**Comment by rossng** [comment]
lgtm

**Pull request merged**